### PR TITLE
kpobozhny. homework04( currency converter + triangle). hackerrank Days08-09

### PR DESCRIPTION
**Pull request by kpobozhny**

### CODE REVIEW

File: homeworks/src/kpobozhny/homework/hw_04/currencyconverter/CurrencyConverter.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    public double convert(double amount) {
+        double converted = new BigDecimal(amount * rate).setScale(2, RoundingMode.UP).doubleValue();
+        return converted;
+        //return Math.round(amount*rate * Math.pow(10, 2)) / Math.pow(10, 2);
+
+    }
+
+    public static void main(String[] args) {
+        CurrencyConverter currencyConverter = new CurrencyConverter(0.037);
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/hw_04/currencyconverter/CurrencyConverter.java:33]: Why rate not 26.68 uah for 1 USD?

File: homeworks/src/kpobozhny/homework/hw_04/triangle/Triangle.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    }
+
+    public Point getVertexC() {
+        return C;
+    }
+
+    public void setVertexC(Point point) {
+        if (((A.getX() == point.getX()) && (B.getX() == point.getX())) ||
+                ((A.getY() == point.getY()) && (B.getY() == point.getY()))) {
```
Comment (PBaranovskyi) [inline homeworks/src/kpobozhny/homework/hw_04/triangle/Triangle.java:63]: Code duplication - you can avoid it if you will store your vertexes in array for example. (Not A,B,C but 1,2,3 and method signature would be not setVertexC(point) but setVertex(index, point))

**Review by PBaranovskyi — CHANGES_REQUESTED** [review]

File: homeworks/src/kpobozhny/homework/hw_04/currencyconverter/CurrencyConverter.java (hunk 1 above)
Comment (kpobozhny) [reply homeworks/src/kpobozhny/homework/hw_04/currencyconverter/CurrencyConverter.java:33]: fixed.

File: homeworks/src/kpobozhny/homework/hw_04/triangle/Triangle.java (hunk 2 above)
Comment (kpobozhny) [reply homeworks/src/kpobozhny/homework/hw_04/triangle/Triangle.java:63]: added new class Triangle2.

**Pull request closed without merging**